src/components/searchResults.js

```diff
@@ -0,0 +1,35 @@
+import propTypes from "prop-types";
+import React from "react";
+import "../styles/searchResults.css";
+
+const SearchResults = ({ results }) => {
+  if (!results.length) {
+    return (
+      <p>
+        We've searched the entirity of an ever expanding universe and found 0
+        results!
+      </p>
+    );
+  } else {
+    return (
+      <div className="search-results">
+        {results.map((result, index) => {
+          return (
+            <img
+              src={result}
+              alt="spaceImage"
```

Comment:
Is there any other data that comes back from the API that could be more useful here so that each image has its own unique `alt` tag?